packages/live-share/src/schema-utils.ts

```diff
@@ -0,0 +1,117 @@
+import {
+    DataObjectTypes,
+    IDataObjectProps,
+} from "@fluidframework/aqueduct";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { LiveDataObject } from "./LiveDataObject";
+import { LiveShareRuntime } from "./LiveShareRuntime";
+import {
+    ContainerSchema,
+    LoadableObjectClass,
+    LoadableObjectClassRecord,
+} from "fluid-framework";
+
+/**
+ * Inject Live Share dependencies into your Fluid container schema.
+ * This should only be done once, right before connecting to a container.
+ * @remarks
+ * Users should not use this method unless you are connecting to a container using `LiveShareClient`.
```

Comment:
Users should not use this method if you are connecting to a container using `LiveShareClient`.